src/VideoPlayer/index.js

```diff
@@ -251,6 +270,30 @@ const videoPlayerPlugin = {
     }
   },
 
+  // open subtitle file
+  // @ params url: subtitle file URL
+  // @ customParser: a customParser to use instead of default parser of the plugin
+  // @ parseOptions.removeSubtitleTextStyles: remove subtitle textstyles possible value true or false
+  // @return parsed subtitles as list of objects
+  openSubtitles(url, customParser = false, options = { removeSubtitleTextStyles: true }) {
```

Comment:
Rather than having a separate function for this, would it make sense to alter the `open` method, and extend it so that you can pass (besides the current aruguments): 
- Either a string with the url of the subtitles (it will use the default parser and default options)
- An object containing a `url`, `customParser` and `options`

That makes it even easier for developers to add subtitles in their apps; they don't have to do:
```js
VideoPlayer.open(url, ....)
VideoPlayer.openSubtitles('my/path/to/subtitles.vtt')
```

but instead

```js
VideoPlayer.open(url, ..., 'my/path/to/subtitles.vtt');
```

Reply:
but If a single video has different language subtitles and wants to switch between languages, then I don't think re-opening the video stream is also required. 
just calling `VideoPlayer.openSubtitles('my/path/to/anotherSubtitles.vtt')` is enough.